src/passes/OptimizeInstructions.cpp

```diff
@@ -459,6 +460,50 @@ struct OptimizeInstructions
         }
         // note that both left and right may be consts, but then we let
         // precompute compute the constant result
+      } else if (binary->op == LtSInt32) {
+        if (auto* c = binary->right->dynCast<Const>()) {
+          auto constValue = c->value.geti32();
+          if (constValue == 0) {
+            // this may increase size for compressed binaries
+            // but it could be faster on some runtimes.
+            // Also it canonical form produced by LLVM codegen
```

Comment:
```suggestion
            // Also it is the canonical form produced by LLVM codegen
```

Also, would you mind reflowing the comments to 80 characters?